src/main/resources/liquibase/master.yml

```diff
@@ -89,3 +89,5 @@ databaseChangeLog:
       - file: liquibase/changelogs/v3_2_0/schema.yml
   - include:
       - file: liquibase/changelogs/v3_3_0/schema.yml
+  - include:
+      - file: liquibase/changelogs/v3_4_0/schema.yml
```

Comment:
3_5_0 ?

Reply:
You're right, this isn't the right version now (it used to be but not anymore).
Feature is now expected in 3.6.0 so I'll move to that version instead.